Projects/Authentication/Sources/OTP/OTPCodeDisplay.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct ShakeEffect: GeometryEffect {
```

Comment:
is there a method to add a geo effect on a view extension so we don't have to call `.modifier` all the time